### PR TITLE
Add nafasebra.md - AboutMe file

### PR DESCRIPTION
**Pull request by nafasebra**

### CODE REVIEW

**Comment by BaseMax** [comment]
```
experiences:
  - company_name: شرکت اول
    role: برنامه نویس بک اند
    description: توضیح پروژه ها و فعالیت
    link: https://github.com/?1
    tags: graphql php c c++ typescript javascript css html
  - company_name: شرکت دوم
    role: برنامه نویس بک اند
    description: توضیح پروژه ها و فعالیت
    link: https://github.com/?2
    tags: graphql php c c++ typescript javascript css html
```

Needs to be updated.

**Review by nafasebra — COMMENTED** [review]
Changed!

**Comment by BaseMax** [comment]
John, please review and check further

Best

File: _networks/nafasebra.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    link: https://asrez.com
+    tags: Html CSS JS ReactJS NextJS TS TailwindCSS
+  - company_name: ShemiranWeb
+    role: Front-end Developer
+    description: In my role at this company, I specialized in crafting website templates using a combination of HTML, CSS, vanilla JavaScript, and TailwindCSS framework, all while closely aligning my work with Figma designs.
+    link: https://shemiranweb.com
+    tags: Html CSS JS TailwindCSS
+---
+
+A passionate self-taught Front-end web developer from Iran. my passion is Implementing the ideas that comes to my brain and take great care in the experience, architecture, and code quality of the things I build.
```
Comment (jbampton) [inline _networks/nafasebra.md]:
```suggestion
A passionate self-taught front-end web developer from Iran. My passion is implementing the ideas that comes to my brain and take great care in the experience, architecture, and code quality of the things I build.
```

**Pull request closed without merging**